ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/pageheader/v1/pageheader/_cq_template.xml

```diff
@@ -21,10 +21,10 @@
     <image
         jcr:primaryType="nt:unstructured"
         jcr:title="Logo"
-        sling:resourceType="wcm/foundation/components/image"
+        sling:resourceType="core/wcm/components/image/v3/image"
```

Comment:
Since you are fixing it here, I am closing this PR, https://github.com/adobe/aem-core-forms-components/pull/773